ui/app/components/secret-version-menu.js

```diff
@@ -1,5 +1,17 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { next } from '@ember/runloop';
 
 export default class SecretVersionMenu extends Component {
+  @service router;
+
   onRefresh() {}
+
+  @action
+  closeDropdown(dropdown) {
+    // strange issue where closing dropdown triggers full transition which redirects to auth screen in production builds
+    // closing dropdown in next tick of run loop fixes it
```

Comment:
🤯

Reply:
Haha yes! I noticed that a full page reload was happening when only a partial transition should have been taking place. I wonder if closing the dropdown in the same tick was throwing an error behind the scenes which caused issues with the transition?